### PR TITLE
Add more check in framebuffer-texture-layer.html

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html [hunk 1]
```diff
@@ -105,6 +105,7 @@
     gl.framebufferTextureLayer(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, tex3d, maxLevels - 1, 0);
     wtu.glErrorShouldBe(gl, gl.NO_ERROR,
         "calling framebufferTextureLayer with an appropriate mipmap level should succeed.");
+    checkFramebuffer([gl.FRAMEBUFFER_INCOMPLETE_ATTACHMENT]);
```
Comment (zhenyao) [inline sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html]: This should be FRAMEBUFFER_COMPELETE, I think.

Comment (kenrussell) [inline sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html]: Should it? I think it should be incomplete. The layer that was attached doesn't exist in the 3D texture.

Comment (qkmiao) [inline sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html]: The attached texture failed completeness check due to texture image doesn't exist in (maxLevels -1) level.

**Comment by zhenyao** [comment]
You are right.  I thought the texture was mipmaped already.

**Pull request merged**